README.md

```diff
@@ -73,6 +73,14 @@ localStorage.setItem('myFirstKey', 'myFirstValue');
 console.log(localStorage.getItem('myFirstKey'));
 ```
 
+### Node ###
+
+Preload `localStorage` at node startup as
+
+```sh
+node -r node-localstorage/register test.js
```

Comment:
Change this to "... /register my-code.js"

Reply:
```suggestion
node -r node-localstorage/register my-code.js
```